src/components/Table/Table.jsx

```diff
@@ -458,10 +459,16 @@ const Table = React.createClass(createLucidComponentDefinition({
 		 * Adjusts the style of the table to have more spacing within the table cells
 		 */
 		hasExtraWhitespace: bool,
+
+		/**
+		 * render the table without borders on the outer edge
+		 */
+		hasNoBorder: bool,
```

Comment:
yes, the css class name is `has-no-border` and it actually removes existing default border styles, so the negation follows suit with that.